connectors/PartnerCenterReferrals/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1097 @@
+        "description": "Get all referrals",
+        "operationId": "GetAllReferrals",
+        "parameters": [
+          {
+            "description": "Expands related entities inline.",
+            "in": "query",
+            "name": "$expand",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Expands related entities inline"
+          },
+          {
+            "description": "Filters the results, based on a Boolean condition.",
+            "in": "query",
+            "name": "$filter",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Filters the results, based on a Boolean condition"
+          },
+          {
+            "description": "Selects which properties to include in the response.",
+            "in": "query",
+            "name": "$select",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Selects which properties to include in the response"
+          },
+          {
+            "description": "Sorts the results.",
+            "in": "query",
+            "name": "$orderby",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Sorts the results"
+          },
+          {
+            "description": "Returns only the first n results.",
+            "format": "int32",
+            "in": "query",
+            "name": "$top",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Returns only the first n results"
+          },
+          {
+            "description": "Skips the first n results.",
+            "format": "int32",
+            "in": "query",
+            "name": "$skip",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "Skips the first n results"
+          },
+          {
+            "description": "Includes a count of the matching results in the response.",
+            "in": "query",
+            "name": "$count",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Includes a count of the matching results in the response"
+          },
+          {
+            "default": "application/json",
+            "description": "Content type of the request.",
+            "in": "header",
+            "name": "content-type",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "description": "The unique identifier to track the request",
+            "in": "header",
+            "name": "MS-CorrelationId",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Correlation Identifier",
+            "x-ms-visibility": "important"
+          },
+          {
+            "default": "60858c1b-2062-40ad-b42c-996b6b5b047d",
+            "description": "Optional, ID to identify the caller - PowerAutomate",
```

Comment:
@didalal Please add a space between Power and Automate like below:

` "description": "Optional, ID to identify the caller - Power Automate",`

Reply:
Done